File: Cmdlets/ExportCultureAndRegionInfoCommand.vb
Imports System.Management.Automation

<Cmdlet("Export", "CultureAndRegionInfo")>
Public Class ExportCultureAndRegionInfoCommand
	Inherits CultureAndRegionInfoCommandBase

	<Parameter(Mandatory:=True)>
	Public Property Name() As String

	<Parameter(Mandatory:=True)>
	Public Property FilePath() As String


	Protected Overrides Sub BeginProcessing()
		CultureAndRegionInfoDictionary(Name).Save(FilePath)
	End Sub
End Class

File: WindowsCmdlets/Commands/ExportCultureAndRegionInfoCommand.cs
using System.Management.Automation;

namespace TIKSN.Cmdlets.Commands
{
    [Cmdlet("Export", "CultureAndRegionInfo")]
    public class ExportCultureAndRegionInfoCommand : CultureAndRegionInfoCommandBase
    {
        [Parameter(Mandatory = true)]
        public string FilePath { get; set; }

        [Parameter(Mandatory = true)]
        public string Name { get; set; }

        protected override void BeginProcessing()
        {
            CultureAndRegionInfoDictionary[Name].Save(FilePath);
        }
    }
}